public/_redirects

```diff
@@ -0,0 +1 @@
+/*    /index.html   200
```

Comment:
Two things:
1. Before merging, let's make sure reasonable cache headers are returned with this response so we don't cache poison the whole domain. I'll do some reading/testing later
2. It's unfortunate that this prevents the server from returning a 404 for a legitimately bad path. So I guess this is actually tradeoff: clean URLs vs clean HTTP status responses.

Reply:
Before merging, let's make sure reasonable cache headers are returned with this response so we don't cache poison the whole domain. I'll do some reading/testing later

Looks good:

```
curl -I https://deploy-preview-24--police-accountability.netlify.app/badurl
HTTP/2 200
cache-control: public, max-age=0, must-revalidate
```